frontEnd/src/main/java/olumofe/raphael/frontEnd/CarController.java

```diff
@@ -0,0 +1,33 @@
+package olumofe.raphael.frontEnd;
+import org.springframework.web.bind.annotation.*;
+import starterProject.App;
+import starterProject.domain.*;
+
+import java.security.Key;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicLong;
+@org.springframework.stereotype.Controller
+
+   @RestController
+public class CarController {
+
+    List<Car> carNewList = Arrays.asList();
+    @PostMapping(path= "/cars",consumes = {"application/json"})
+    @ResponseBody
+    public Map<String,Car> createCarList(@RequestBody List<Car> carList){
+       carNewList = carList;
+        HashMap<String,Car> map = new HashMap<String, Car>();
```

Comment:
Type should be Map not HashMap. It doesn't particularly matter in this instance, but you should get into the habit of using it.

https://stackoverflow.com/questions/9852831/polymorphism-why-use-list-list-new-arraylist-instead-of-arraylist-list-n